### PR TITLE
Registration confirmation message

### PR DESCRIPTION
**Pull request by Julesssss**
Add confirmation text to registration password confirmation dialog

### CODE REVIEW

**Comment by nbau21** [comment]
+1 easy peasy

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -11,6 +11,7 @@
     <string name="action_sign_in_short">Sign in</string>
     <string name="action_cancel">Cancel</string>
     <string name="action_ok">OK</string>
+    <string name="registration_confirmation">Note that because your notes are encrypted on your device using you password, Standard Notes does not have a password reset option. You can not forget your password.</string>
```
Comment (mobitar) [inline app/src/main/res/values/strings.xml]: > using you password

;)

**Comment by herbig** [comment]
Also can not

**Comment by Julesssss** [comment]
Thanks @herbig. @mobitar - 'Cannot' seems to be the grammatically correct option, perhaps iOS needs to align with our updated string. 

> Note that because your notes are encrypted on your device using your password, Standard Notes does not have a password reset option. You cannot forget your password.

**Comment by mobitar** [comment]
Thanks, fixed on iOS.

**Comment by herbig** [comment]
Ha, yeah didn't mean to be so short with my grammar message, was walking my dog so typing on mobile.

**Comment by nbau21** [comment]
^ or maybe it was your dog commenting on the grammar ;)

**Comment by mobitar** [comment]
12 characters. Seems trainable.

**Pull request closed without merging**